backend/mail-service/src/main/kotlin/mail/services/MailService.kt

```diff
@@ -106,6 +106,63 @@ class MailService(
         """.trimIndent()
     }
 
+    private fun addSupportTemplate(text: String): String {
+        return """
+        <!DOCTYPE HTML>
+        <html>
+            <body style='margin:0; padding:0; font-family: "IBM Plex Sans", sans-serif, system-ui, -apple-system,
+                Segoe UI, Roboto, Ubuntu, Cantarell, Oxygen, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+                "Segoe UI Symbol", "Noto Color Emoji";'>
+                <div style="padding: 2em 4em 2em 4em; max-width: 600px; margin-left: auto; margin-right: auto;">
+                    $text
+                </div>
+            </body>
+        </html>
+        """.trimIndent()
+    }
+
+    suspend fun sendSupportTicket(
+        fromEmail: String,
+        subject: String,
+        text: String
+    ) {
+        val recipientAddress = InternetAddress("support@escience.sdu.dk")
+
+        // Setup mail server
+        val properties = System.getProperties()
+        properties.setProperty("mail.smtp.host", "localhost")
+        properties.setProperty("mail.smtp.port", "25")
```

Comment:
This is done in both this function and `send`. Couldn't this be done on service initialization or is it a per request thing?